package.json

```diff
@@ -71,9 +81,11 @@
     "eslint": "^7.19.0",
     "eslint-config-prettier": "^7.2.0",
     "eslint-plugin-prettier": "^3.3.1",
+    "glob": "7.1.6",
     "graphql": "^15.5.0",
     "jest": "^26.6.3",
     "prettier": "^2.2.1",
+    "replacestream": "4.0.3",
```

Comment:
Please dont install deps with exact versions.

Reply:
A sorry! I tend to pin them and let a bot update the pinned version for dev dependencies. WIll fix

Reply:
I want to add the Dependabot too; however, [Yarn 2 support is still in works](https://github.com/dependabot/dependabot-core/issues/1297)...

Reply:
You should try to use renovate, I prefer it over dependabot. haven't used it with yarn2 though as I don't like it :D